.grit/patterns/use-exponentiation-operator.md

```diff
@@ -9,7 +9,25 @@ ES7 introduced the exponentiation operator `**` so that using `Math.pow` is no l
 tags: #ES7, #SE
 
 ```grit
-`Math.pow($base, $exponent)` => `$base ** $exponent`
+engine marzano(0.1)
+language js
+
+`Math.pow($base, $exponent)` as $expression where {
+    $args = [],
+    if (! $base <: number()) {
+        $args += `($base)`
+    } else {
+        $args += $base
+    },
+    if (! $exponent <: number()) {
+        $args += `($exponent)`
+    } else {
+        $args += $exponent
+    },
+    $separator = ` ** `,
+    $newExpression = join(list = $args, $separator),
+    $expression => `$newExpression`
+}
```

Comment:
This is clunky, but the best I could do in my experimentation without linearization.